graphql_compiler/compiler/emit_sql.py

```diff
@@ -103,6 +103,25 @@ def _find_folded_outputs(ir):
     return folded_outputs
 
 
+def _find_x_count_filters(ir):
+    """For each fold path, find all filters involving _x_count."""
+    _x_count_filters = []
+    global_operation_start = False
+    for block in ir.ir_blocks:
+        # _x_count filters only occur after encountering a GlobalOperationStart block.
+        if isinstance(block, blocks.GlobalOperationsStart):
+            global_operation_start = True
+        # Check the left and right side of Filter predicates to see if _x_count is involved.
+        if global_operation_start and isinstance(block, blocks.Filter):
+            if (isinstance(block.predicate.left, FoldedContextField) and
+                    block.predicate.left.fold_scope_location.field == '_x_count'):
```

Comment:
Do we have a constant we could refer to for `_x_count`? If not, can we make one?